src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      method="get"
+      data-qa="big"
+    >
+      <label class="search-label">
+        <input
+          type="text"
+          name="input"
+          class="input"
```

Comment:
Consider giving the input elements unique name attributes to differentiate them, for example, 'inputBig' and 'inputSmall'.